OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/InAppMessagesManager.kt

```diff
@@ -182,7 +188,7 @@ internal class InAppMessagesManager(
         }
 
         suspendifyOnThread {
-            fetchMessages()
+            fetchMessages(null)
```

Comment:
This could result in skipping an offset we should be waiting on, we should pass the `offset` value in here.